packages/cf-component-form/src/FormHeader.js

```diff
@@ -1,19 +1,23 @@
 import React, { PropTypes } from 'react';
+import { Heading } from 'cf-component-heading';
+import { createComponent } from 'cf-style-container';
 
-class FormHeader extends React.Component {
-  render() {
-    return (
-      <div className="cf-form__header">
-        <h3 className="cf-form__title">
-          {this.props.title}
-        </h3>
-      </div>
-    );
-  }
-}
+const styles = ({ theme }) => ({
+  padding: theme.padding,
+  borderBottom: theme.borderBottom
+});
+
+const FormHeader = ({ className, title }) => (
+  <div className={className}>
+    <Heading size={4}>
```

Comment:
Because it's more similar to the old version where we remove margin when used inside of form (we should not do stuff like this) but our headings come with fixed margin. We most likely make more style changes pretty soon since we don't really use cf-component-form as we should.